hybridse/src/codegen/udf_ir_builder_test.cc

```diff
@@ -115,6 +115,10 @@ TEST_F(UdfIRBuilderTest, dayofweek_date_udf_test) {
     Date date(2020, 05, 22);
     CheckUdf<int32_t, Date>("dayofweek", 6, date);
 }
+TEST_F(UdfIRBuilderTest, dayofyear_date_udf_test) {
+    Date date(2020, 05, 22);
+    CheckUdf<int32_t, Date>("dayofyear", 143, date);
```

Comment:
Please add cases for:
- the first day of a year like 2021-01-01
- the last day of a year like 2021-12-31
- the last day of a leap year like 2020-12-31

Reply:
good idea!

Reply:
One of the previous test cases is failing, I'll try to figure out why